test/verify/test_roialign_nonstandard.cpp

```diff
@@ -0,0 +1,36 @@
+
+#include "verify_program.hpp"
+#include <migraphx/program.hpp>
+#include <migraphx/generate.hpp>
+#include <migraphx/make_op.hpp>
+
+struct test_roialign_nonstandard : verify_program<test_roialign_nonstandard>
+{
+    migraphx::program create_program() const
+    {
+        migraphx::program p;
+        auto* mm = p.get_main_module();
+        auto x_s = migraphx::shape::from_permutation(
```

Comment:
Should we add test for slice and broadcast as well ?

Reply:
I dont think its necessary, I dont know what other reviewers think.

Reply:
@scxiao Do you think we will encounter slice or broadcast for ROIAlign ? I doubt it.

Reply:
Well using a transpose is enough of a test to ensure we are using the shape class to compute the indices. We dont have special cases for transpose, slice, or broadcasted gpu kernels.